lib/entities/parse_address_from_domain.dart

```diff
@@ -44,10 +44,15 @@ class AddressResolver {
       if (text.startsWith('@') && !text.substring(1).contains('@')) {
         final formattedName = text.substring(1);
         final twitterUser = await TwitterApi.lookupUserByName(userName: formattedName);
-        final address = extractAddressByType(
+        final addressFromBio = extractAddressByType(
             raw: twitterUser.description ?? '', type: CryptoCurrency.fromString(ticker));
-        if (address != null) {
-          return ParsedAddress.fetchTwitterAddress(address: address, name: text);
+        final addressFromPinnedTweet = extractAddressByType(
+            raw: twitterUser.pinnedTweet ?? '', type: CryptoCurrency.fromString(ticker));
```

Comment:
We can move this after the `if (addressFromBio != null)` as it won't have to look through the pinned tweets if it found an address in the Bio